src/main/webapp/app/core/user/settings/passkey-settings/util/bitwarden.util.ts

```diff
@@ -37,27 +36,24 @@ export function getCredentialFromMalformedBitwardenObject(malformedBitwardenCred
         return null;
     }
 
-    const clonedCredential: Credential = cloneDeep(malformedBitwardenCredential) as unknown as Credential;
-    const credential = JSON.parse(JSON.stringify(clonedCredential));
```

Comment:
I did not refactor that part properly, this was just an attempt to easily get a serializable version of the malformed bitwarden credential